package.json

```diff
@@ -249,7 +249,7 @@
     "@types/react-dom": "18.0.11"
   },
   "engines": {
-    "node": ">=16"
+    "node": ">=16.8.0"
```

Comment:
I realized that `webcrypto` was actually added back in `15.0.0` https://nodejs.org/dist/latest-v18.x/docs/api/crypto.html#cryptowebcrypto

But we might still want to bump this version since `undici` (what we use in App Router) requires `16.8.0`.

Let me know if this should be a different PR.